runtime/calamari/src/xcm_config.rs

```diff
@@ -360,5 +360,6 @@ impl orml_xtokens::Config for Runtime {
     type MaxAssetsForTransfer = MaxAssetsForTransfer;
     type MinXcmFee = AssetManager;
     type MultiLocationsFilter = AssetManager;
+    type OutgoingAssetsFilter = ();
```

Comment:
I would think we would want filter implemented for calamari too

Reply:
No, this is just a temporary thing for MANTA, and for emergencies in the future.

Reply:
Does `()` always return true?